tdp/cli/commands/browse.py

```diff
@@ -35,20 +41,27 @@
 )
 @database_dsn
 def browse(
-    deployment_id: int, operation: str, limit: int, offset: int, database_dsn: str
+    deployment_id: int, operation: str, plan: bool,limit: int, offset: int, database_dsn: str
```

Comment:
apply file autoformatting before committing. on my Mac it is `Shift + Option + F`.

Reply:
Done.